data/whispering void/critters.txt

```diff
@@ -60,7 +60,7 @@ ship "Pincer Beast"
 	description `This creature appears to have an outer shell made of small asteroids held together by strands of carbon nanotubes. It pulls rocks through arms into its main cavity where it processes them to make more shell. The multi-colored hairs pointing back seem to be a drive system of sorts, manipulating the gravity field around the creature to accelerate it. There is no apparent hyperdrive mechanism, but somehow it is able to open a hyperspace window.`
 
 effect "pincer beast shatter"
-	sprite "effect/pincer beast shatter"
+	sprite "effect/pincer beast-shatter"
```

Comment:
If this is actually the name of the sprite, I think perhaps *that* is what should be changed, not this line.